cmd/wallet-web/src/components/Vaults/VaultCard.vue

```diff
@@ -1,36 +1,42 @@
 <template>
-  <router-link
+  <div
```

Comment:
Switching root `router-link` back to `div` makes some part of the `VaultCard` unclickable. I have read some articles and it looks like the best way to implement _breakout_ links or buttons would be to insert a pseudo element. See https://css-tricks.com/breakout-buttons/. Basically whenever we cannot go without placing `div`s inside links or buttons, we should instead place them inside and add pseudo element to expand the clickable area to the whole card. Let's add this approach here.

Reply:
I have created separate issue #1286 , since making the whole vault card router link was interring with the vault flyout click event. We can revisit this solution